Formula/qt.rb

```diff
@@ -64,6 +64,10 @@ class Qt < Formula
   uses_from_macos "libxslt"
   uses_from_macos "zlib"
 
+  on_macos do
+    depends_on "molten-vk"
```

Comment:
```suggestion
    depends_on "molten-vk" => [:build, :test]
```
Then add generic dependencies:
```ruby

    depends_on "vulkan-headers" => [:build, :test]
    depends_on "vulkan-loader" => :test
```
On linux we can use vulkan icd from `mesa`.